spec/helpers/hyrax_helper_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rspec'
```

Comment:
This require should be `rails_helper` instead of `rspec`.  I think that is the source of the failing circleci build.